deployment/docker/README.md

```diff
@@ -189,6 +189,37 @@ Please see more examples on integration of VictoriaLogs with other log shippers
 * [fluentd](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/victorialogs/fluentd)
 * [datadog-serverless](https://github.com/VictoriaMetrics/VictoriaMetrics/tree/master/deployment/docker/victorialogs/datadog-serverless)
 
+## VictoriaTraces server
+
+To spin-up environment with VictoriaTraces run the following command:
+```
```

Comment:
We should also mention that the repo has to be cloned and a specific branch checked out

```
git clone -b victoriametrcs --single-branch https://github.com/VictoriaMetrics/VictoriaMetrics.git
cd VictoriaMetrics
```

Reply:
ToC updated